.prettierrc.mjs

```diff
@@ -0,0 +1,6 @@
+export default {
+  tabWidth: 2,
+  useTabs: false,
+  trailingComma: "es5",
+  plugins: ["prettier-plugin-ember-template-tag"]
```

Comment:
you're not running prettier on configs themselves? 😛 

```suggestion
  plugins: ["prettier-plugin-ember-template-tag"],
```

Reply:
Added a few more file extensions, so we're covered